create.js

```diff
@@ -34,7 +34,22 @@ module.exports = function (path, opts, keys) {
   var _get = db.get
   var _del = db.del
 
-  db.del = (key, cb) => {
+  const deleteFeed = (feed, cb) => {
+    pull(
+      db.createUserStream({ id: feed }),
+      pull.asyncMap((msg, cb) => {
+        const key = msg.key
+
+        deleteMessage(key, (err) => {
+          cb(err, key)
+        })
+      }),
+      pull.filter(),
```

Comment:
Why is this needed?

Reply:
It was added because sometimes we were getting `null`, but that could've been a Patchwork quirk. I'll try removing it and see what happens.

Reply:
Could also just put a comment `// removes nulls`